dash_reusable_components.py

```diff
@@ -218,9 +220,17 @@ def NamedInlineRadioItems(name, short, options, val, **kwargs):
 
 
 # Custom Image Components
-def InteractiveImagePIL(image_id, image, enc_format='png', display_mode='fixed', dragmode='select', verbose=False, **kwargs):
+def InteractiveImagePIL(image_id,
+                        image,
+                        enc_format='png',
+                        display_mode='fixed',
+                        dragmode='select',
+                        verbose=False,
+                        **kwargs):
     if enc_format == 'jpeg':
-        encoded_image = pil_to_b64(image, enc_format=enc_format, verbose=verbose, quality=95)
+        if image.mode == 'RGBA':
+            image = image.convert('RGB')
+        encoded_image = pil_to_b64(image, enc_format=enc_format, verbose=verbose, quality=80)
```

Comment:
I think this line is too long for PEP8; try putting each argument on its own line, like you did for `InteractiveImagePIL`  above